spyder/preferences/languageserver.py

```diff
@@ -105,124 +105,128 @@ class LSPServerEditor(QDialog):
     NON_EMPTY_REGEX = re.compile(r'^\S+$')
     JSON_VALID = _('JSON valid')
     JSON_INVALID = _('JSON invalid')
+    MIN_SIZE = QSize(800, 600)
 
     def __init__(self, parent, language=None, cmd='', host='127.0.0.1',
                  port=2084, args='', external=False, configurations={},
                  **kwargs):
         super(LSPServerEditor, self).__init__(parent)
+
+        description = _(
+            'To create a new configuration, you need to select a programming '
+            'language, along with a executable name for the server to '
+            'execute (If the instance is local), and the host and port. '
+            'Finally, you need to provide the arguments that the server '
+            'accepts. <br><br>The placeholders <tt>{host}</tt> and '
+            '<tt>{port}</tt> refer to the host and the port, respectively.'
```

Comment:
@CAM-Gerlach, please take a look at this text to see if we can improve it.

Reply:
Yes, it is pretty good overall but there is some room for improvement. Revised:

```
            'To create a new server configuration, you need to select a '
            'programming language and the server hostname/IP and port to use. '
            'If starting a local server, also provide the command to execute it '
            'and any command-line that should be passed to the it on startup.'
```